src/Php/PhpVersions.php

```diff
@@ -28,4 +28,9 @@ public function producesWarningForFinalPrivateMethods(): TrinaryLogic
 		return IntegerRangeType::fromInterval(80000, null)->isSuperTypeOf($this->phpVersions)->result;
 	}
 
+	public function socketCreateUsesResource(): TrinaryLogic
+	{
+		return IntegerRangeType::fromInterval(null, 79999)->isSuperTypeOf($this->phpVersions)->result;
```

Comment:
I'd write this as "since 8.0" rather than until "7.9999". So rename this method to what it actually returns since 8.0.